.github/workflows/build.yml

```diff
@@ -226,6 +226,13 @@ jobs:
         python3 -c 'import pyxrootd; print(pyxrootd)'
         python3 -c 'from XRootD import client; print(client.FileSystem("root://someserver:1094"))'
 
+    - name: Build sdist using publishing workflow
+      run: |
+        . /opt/rh/devtoolset-7/enable
+        cp packaging/wheel/* .
+        ./publish.sh
+        ls -lhtra dist/
```

Comment:
@simonmichal I added in this chunk into the CI after the RPM build so that your comment is now tested against in the case of the RPM install.